pizza_cutter/slice_utils/interpolate.py

```diff
@@ -74,7 +77,138 @@ def _interp_patch(*, image, bad_msk, i, j, size, buff):
         return None
 
 
+@njit
+def _get_nearby_good_pixels(image, bad_msk, buff=4):
+    """
+    get the set of good pixels surrounding bad pixels.
+
+    Parameters
+    ----------
+    image: array
+        The image data
+    bad_msk: bool array
+        2d array of mask bits.  True means it is a bad
+        pixel
+
+    Returns
+    -------
+    bad_pix:
+        bad pix is the set of bad pixels, shape [nbad, 2]
+    good_pix:
+        good pix is the set of bood pixels around the bad
+        pixels, shape [ngood, 2]
+    good_im:
+        the set of good image values, shape [ngood]
+    good_ind:
+        the 1d indices of the good pixels row*ncol + col
+    """
+
+    nrows, ncols = bad_msk.shape
+
+    ngood = bad_msk.size*3
```

Comment:
This is like 3x the full image size, right? I think we only need something like `np.sum(bad_msk)*16` or something close to that.

Reply:
Note `np.sum(bad_msk)` is the number of bad pixels.

Reply:
good point, thanks